modules/new_profile/php/NDB_Form_new_profile.class.inc

```diff
@@ -297,6 +298,14 @@ class NDB_Form_New_Profile extends NDB_Form
                     $errors['PSCID'] = 'PSCID has already been registered';
             }
         }
+
+        // validate site entered
+        $site = $values['psc'];
+
+        if (empty($site) || !$user->hasCenter($site)) {
+            $errors['psc'] = "Site must be selected from the available dropdown.";
+        }
+
```

Comment:
@ridz1208 

do we need this here? check line 228 above, currently site is required and you can not create one without setting it.

Reply:
back-end validation is always a plus